types/appium-adb/index.d.ts

```diff
@@ -0,0 +1,7 @@
+// Type definitions for appium-adb 8.16
+// Project: https://github.com/appium/appium-adb
+// Definitions by: Tiger Oakes <https://github.com/NotWoods>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export * from './lib/adb';
+export { default } from './lib/adb';
```

Comment:
```suggestion
import {
  ADB,
  DEFAULT_ADB_PORT,
  getAndroidBinaryPath,
  getSdkRootFromEnv
} from './lib/adb';

export default ADB;
export { ADB, DEFAULT_ADB_PORT, getAndroidBinaryPath, getSdkRootFromEnv };
```
notice, this is just same as in source (ok, nearly)

Reply:
Is it preferred to not export the interfaces?

Reply:
oh, no, the only concern here was to not export everything, say types that are not needed to use package itself and are added by TS migration. So it's perfectly ok to export types e.g. that way:
```ts
// Type definitions for appium-adb 8.16
// Project: https://github.com/appium/appium-adb
// Definitions by: Tiger Oakes <https://github.com/NotWoods>
// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped

import {
    ADB,
    DEFAULT_ADB_PORT,
    getAndroidBinaryPath,
    getSdkRootFromEnv,
    AdbExecutable,
    CreateAdbOptions,
} from './lib/adb';

export default ADB;
export {
    ADB, 
    DEFAULT_ADB_PORT,
    getAndroidBinaryPath,
    getSdkRootFromEnv,
    AdbExecutable,
    CreateAdbOptions
};
```
(list every interface once can find useful or even required when using those types).
Sorry for the hassle. 
I"ll accept, you're free to amend, thx!

Reply:
Updated so the index file imports from tools directly like this!